src/cloud/components/ApplicationPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropsWithChildren } from 'react'
+import { TopbarPlaceholder } from '../../design/components/organisms/Topbar'
+import ContentLayout, {
+  ContentLayoutProps,
+} from '../../design/components/templates/ContentLayout'
+
+type ApplicationPageProps = {
+  topbarPlaceholder?: boolean
```

Comment:
Just a component with the same height. Making it so that content pages without any topbar ( error pages etc.. ) are still vertically aligned with the other pages.